### ISSUE TITLE
Error when open transaction details screen

### ISSUE THREAD

**Issue by churik** [status-im/status-react#8108]
### Description
_Type_: Bug
_Summary_: opening transaction details screen from `Transaction history` causes an error.

#### Expected behavior
No error

#### Actual behavior
![photo!UNITO-UNDERSCORE!2019-05-02 13 38 50](https://user-images.githubusercontent.com/4557972/57072916-a26bee80-6cdf-11e9-81b5-86e0a0ab9b42.jpeg)

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

* Open Status
* Restore any account with transactions
* Open Transaction History
* Check Transaction detail by tapping on any transaction

### Additional Information

* Status version: nightly 02/05/2019
* Operating System: Android, iOS

#### Logs
[comment]: # (if on Android please replicate bug whilst running adb logcat)

```
05-02 13:37:14.307 1065 1406 V APM_AudioPolicyManager: setOutputDevice() setting same device 0x0000 or null device
05-02 13:37:14.318 30725 30767 E unknown:ReactNative: Unknown transaction type:
```

[log.log](https://github.com/status-im/status-react/files/3137831/log.log)

**Comment by annadanchenko** [comment]
@flexsurfer @mandrigin can you help to investigate which change causes this issue, please?

**Comment by mandrigin** [comment]
@annadanchenko @flexsurfer I can take a look in ~2h, if anyone has time before that feel free to take it over (but ping me here).

**Comment by churik** [comment]
Most likely it is introduced by #8050 - checked PRs list for that day when issue was introduced and IMO #8050 or TTT PR can cause it.
Checked both PRs and issue is reproducible only in #8050.

**Comment by mandrigin** [comment]
if so, @flexsurfer can you take a look? it would be much easier for you to fix it...

**Comment by asemiankevich** [comment]
can't reproduce in nightly https://ci.status.im/job/status-react/job/manual/307/console (nightly with new status go on staging fleet)

**Comment by flexsurfer** [comment]
@churik could you try please in this PR https://github.com/status-im/status-react/pull/8140

**Comment by churik** [comment]
closed in #8140
